src/Plugin/Validation/Constraint/ValidOgMembershipReferenceConstraint.php

```diff
@@ -23,4 +23,11 @@ class ValidOgMembershipReferenceConstraint extends Constraint {
    */
   public $notValidGroup = 'The entity %label is not defined as a group.';
 
+  /**
+   * Not a valid group message.
+   *
+   * @var string
+   */
+  public $NotAllowedToPostInGroup = 'You are not allowed to post content in the group %label';
```

Comment:
Class property $NotAllowedToPostInGroup should use lowerCamel naming without underscores

```suggestion
  public $notAllowedToPostInGroup = 'You are not allowed to post content in the group %label';
```